### PR TITLE
Newsletter signup on blog

### PR DESCRIPTION
**Pull request by Lukewh**
Fixes https://github.com/canonical-websites/snapcraft.io/issues/1807

## QA
- Pull branch
- `./run`
- Visit http://0.0.0.0:8004/blog or https://snapcraft-io-canonical-websites-pr-1809.run.demo.haus/blog
- Sign up to the newsletter - you should see a "subscibed" message next to the button

### CODE REVIEW

**Comment by bartaz** [comment]
Something for later maybe, but it would be good if any kind of email notification (or ideally verification email) would be sent.
Right now apart from 'subscribed' text showing up there is nothing in email confirming that it was successful. Also it feels like anyone can put anyones email there and subscribe them to blog without confirmation.

File: static/js/public/snap-details/reportSnap.js [hunk 1]
```diff
@@ -36,17 +37,6 @@ function showError(modal) {
   showEl(modal.querySelector(".js-report-snap-error"));
 }
 
-function buttonLoading(button) {
```
Comment (bartaz) [inline static/js/public/snap-details/reportSnap.js]: Cool that we can reuse that 👍

**Comment by Lukewh** [comment]
> Something for later maybe, but it would be good if any kind of email notification (or ideally verification email) would be sent.
> Right now apart from 'subscribed' text showing up there is nothing in email confirming that it was successful. Also it feels like anyone can put anyones email there and subscribe them to blog without confirmation.

Yeah, I think once we have this signup in place, we should definitely make the experience tighter and nicer. I'm not sure what the limitations are with Marketo but I'll have a chat with Lucia and see what she says :+1:

File: webapp/blog/views.py [hunk 2]
```diff
@@ -44,8 +45,15 @@ def homepage():
         filter = None
 
     try:
+        get_page = 1
```
Comment (bartaz) [inline webapp/blog/views.py]: A little comment here would be nice to explain that we offset because of newsletter card

Comment (Lukewh) [reply webapp/blog/views.py]: Ah, thought I'd added one :+1:

**Pull request merged**